tools/respecDocWriter.js

```diff
@@ -96,6 +96,19 @@ const tasks = {
         const msg = `HTTP Error ${response.code}: ${url}`;
         throw new Error(msg);
       }
+      const isRespecDoc = yield nightmare
+        .wait(function(){
+          return document.readyState === "complete";
+        }) 
+        .evaluate(function(){
+          const selector = "script[src='https://www.w3.org/Tools/respec/respec-w3c-common']";
```

Comment:
can't you test the presence of a ReSpec global (assuming there is still such a thing)? 

Relying the the URL detection makes the tool more brittle (I know I've used it on locally hosted version of respec for instance)